images/linux/scripts/base/repos.sh

```diff
@@ -9,7 +9,11 @@ LSB_RELEASE=$(lsb_release -rs)
 # Install Microsoft repository
 wget https://packages.microsoft.com/config/ubuntu/$LSB_RELEASE/packages-microsoft-prod.deb
 dpkg -i packages-microsoft-prod.deb
```

Comment:
Repo `https://packages.microsoft.com/repos/microsoft-ubuntu-bionic-prod bionic main` - already present in packages-microsoft-prod.deb